src/test/java/com/google/devtools/build/lib/runtime/CommandLinePathFactoryTest.java

```diff
@@ -0,0 +1,58 @@
+package com.google.devtools.build.lib.runtime;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.devtools.build.lib.vfs.PathFragment;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for {@link CommandLinePathFactory}. */
+@RunWith(JUnit4.class)
+public class CommandLinePathFactoryTest {
+  @Test
+  public void createFromAbsolutePath() {
+    CommandLinePathFactory factory = new CommandLinePathFactory(ImmutableMap.of());
+
+    assertThat(factory.create("/absolute/path/1"))
+        .isEqualTo(PathFragment.create("/absolute/path/1"));
+    assertThat(factory.create("/absolute/path/2"))
+        .isEqualTo(PathFragment.create("/absolute/path/2"));
+  }
+
```

Comment:
Add a test to verify that the empty path is rejected.

Reply:
done